In Python 3.9, it doesn't seem to pass the mypy check in pre-commit

**Describe the bug** According to the CONTRIBUTING.md,

>Development on the latest version of Python is preferred.  As of this writing it's 3.9.

But I ran the `pre-commit run -a` command and I was faced with mypy errors.

```
$ pre-commit run -a
black....................................................................Passed
flake8...................................................................Passed
mypy.....................................................................Failed
- hook id: mypy
- exit code: 2

src/black_primer/lib.py:37:12: error: syntax error in type comment
src/black_primer/lib.py:38:22: error: syntax error in type comment
src/black_primer/lib.py:42:10: error: syntax error in type comment
src/black_primer/lib.py:44:10: error: syntax error in type comment
src/black_primer/lib.py:45:10: error: syntax error in type comment
src/black_primer/lib.py:46:6: error: syntax error in type comment
src/black_primer/lib.py:111:38: error: syntax error in type comment
src/black_primer/lib.py:155:21: error: syntax error in type comment
src/black_primer/lib.py:159:6: error: syntax error in type comment
src/black_primer/lib.py:188:38: error: syntax error in type comment
src/black_primer/lib.py:213:6: error: syntax error in type comment
src/black_primer/lib.py:230:13: error: syntax error in type comment
Found 12 errors in 1 file (checked 14 source files)

prettier.................................................................Passed
```

**To Reproduce** The following commands:

```
# At the root of this repository
python3.9 -m venv env
. env/bin/activate
pip install -U pip
pip install pipenv
pipenv install --dev
pipenv shell
pre-commit install
pre-commit run -a
# See the above mypy errors
```

**Expected behavior** no mypy errors.

**Environment:**

- Version: master (a522aa4)
- OS and Python version: macOS/Python 3.9.0
- mypy version

```
$ mypy --version
mypy 0.790
$ pipenv run mypy --version
mypy 0.790
```

**Does this bug also happen on master?** Yes

**Additional context** 
I think others who want to contribute Black will have trouble.
As a temporary workaround, no error occurs in using Python **3.8**.

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -18,7 +18,7 @@ repos:
         additional_dependencies: [flake8-bugbear]
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.770
+    rev: v0.780
     hooks:
       - id: mypy
         exclude: ^docs/conf.py

--- a/src/black_primer/lib.py
+++ b/src/black_primer/lib.py
@@ -59,10 +59,12 @@ async def _gen_check_output(
         raise
 
     if process.returncode != 0:
+        returncode = process.returncode
+        if returncode is None:
+            returncode = 69
+
         cmd_str = " ".join(cmd)
-        raise CalledProcessError(
-            process.returncode, cmd_str, output=stdout, stderr=stderr
-        )
+        raise CalledProcessError(returncode, cmd_str, output=stdout, stderr=stderr)
 
     return (stdout, stderr)
 

